src/PrMerger.js

```diff
@@ -145,8 +145,9 @@ class PrMerger {
 // PrMerger's create and execute() wrapper.
 async function Step() {
     let mergerer = new PrMerger();
-    return mergerer.execute();
-};
+    const result = await mergerer.execute();
```

Comment:
I do not think this `await` was missing -- the old code returned a promise, which any async call is allowed to do. What is probably not needed here is the `async` designation -- this function is not an async function. I suggest restoring the old no-`await` code, removing `async`, and changing the Step() description to something like

```
// promises to process all PRs once, hiding PrMerger from callers
```